src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -2009,41 +2629,70 @@ public enum RisingRivals implements LogicCardInfo {
             energyCost ()
             attackRequirement {}
             onAttack {
-              damage 0
+              flip 2, {
+                def pcsList = my.all.findAll{it.numberOfDamageCounters}
+                if(pcsList) {
+                  heal 10, pcsList.select("Remove 1 damage counter from 1 of your Pokémon")
+                }
+              }
```

Comment:
You're supposed to heal up to two damage from a single Pokémon, you should not be able to remove a damage counter from two different Pokémon. That is my understanding of the card text.